handlers/common.go

```diff
@@ -31,10 +32,58 @@ func PingNoOp(event *events.Event, apiClient *client.RancherClient) error {
 	return nil
 }
 
+func acquire_networked_global_persistent_lock(resourceId string) error {
+	return nil
+}
+
+func release_lock(lock string) {
+	return
+}
+
 func getMachineDir(machine *client.Machine) (string, error) {
 	return getBaseMachineDir(machine.ExternalId)
 }
 
+func sync_resource(machine *client.Machine) error {
```

Comment:
The more I think about it, I actually really like the idea of extracting the config on every update. This makes normal operation and operation after a failure (ie loss of disk/start new gms) exactly the same.